### DISCUSSION TITLE
Searching for method pump.initTimer()

### DISCUSSION THREAD

**Discussion by Rovagoo1** [Zer0-bit/gaggiuino#538] [General]
Hi,
in gaggiuino/src/peripherals/pump.cpp the method getCPS() uses another method pump.initTimer(). Thought I would find initTimer() in the correct PSM.cpp from banoz, but I can't find it here. Where do I search for this method or is it not relevant in this context?
VG Robert

**Comment by banoz** [comment]
make sure you're looking into correct branch
